src/services/fetch/types.ts

```diff
@@ -1,7 +1,14 @@
 import { TimeString } from '@types';
 
-export type RequestInit = globalThis.RequestInit & { timeout?: TimeString };
+export type RequestInit = globalThis.RequestInit & {
+  timeout?: TimeString;
+  retries?: number;
+  retryDelay?: number;
+  retryWithTimeout?: boolean;
```

Comment:
I think it would be a little clearer to call it `retryIfTimeouted` or `retryWhenTimeouted`. Because I though it meant that the retry would also have a timeout or something like that